test/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanDaoMockTest.java

```diff
@@ -184,12 +209,84 @@ public void updateSchedulePlanWritesCriteria() {
     }
     
     @Test
-    public void deleteSchedulePlanDeletesCriteria() {
+    public void createSchedulePlanCannotCreatedDeletedPlan() {
+        SchedulePlan plan = SchedulePlan.create();
+        plan.setDeleted(true);
+        
+        dao.createSchedulePlan(TestConstants.TEST_STUDY, plan);
+        
+        verify(mapper).save(schedulePlanCaptor.capture());
+        assertFalse(schedulePlanCaptor.getValue().isDeleted());
+    }
+    
+    @Test
+    public void getSchedulePlansIncludeDeleted() {
```

Comment:
Nitpick: This test and the next test have their names reversed.

Reply:
Swapping test names